### PR TITLE
Blogs part 3

### PR DESCRIPTION
**Pull request by jhinkey**
Here you go. :-)

You can view it in the dev-uat knowledge base: https://dev-uat.liferay.com/discover/portal/-/knowledge_base/7-0/publishing-blogs

https://issues.liferay.com/browse/LRDOCS-1906

### CODE REVIEW

File: discover/portal/articles/07-collaborating/01-publishing-blogs.markdown [hunk 1]
```diff
@@ -11,9 +11,9 @@ Its editor delivers features you'd expect, in an elegant in-context manner. It
 has a complete set of WYSIWYG controls that appear where and when you need them.
 You can stay in this easy-to-use mode or switch to source mode to edit your
 content's HTML code. In source mode, you can work with light text on a dark
-background or dark text on a light background. To cap things off, you can do
-fullscreen editing to minimize distractions as your create your posts. You'll
-have a satisfying experience creating awesome blog posts. 
+background or dark text on a light background. To cap things off, you can
+enlarge the HTML editor window to minimize distractions as you create your
```
Comment (sez11a) [inline discover/portal/articles/07-collaborating/01-publishing-blogs.markdown]: Isn't the main benefit of this not the enlarging of the window, but the dual pane that shows the HTML on the left and the preview on the right?

File: discover/portal/articles/07-collaborating/01-publishing-blogs.markdown [hunk 2]
```diff
@@ -66,31 +68,31 @@ add rows, columns, and cells.
...
+If you want to minimize distractions as you edit your blog entry's HTML code,
+we've got you covered. You can click the *Enlarge* icon
+(![Enlarge](../../images/enlarge-icon.png)) to open the HTML editor in a
+full-size *Edit Content* window. The first icons above the editor allow you to
+arrange the HTML editor and rendered content area, horizontally or vertically.
+These dual views allow you to edit the HTML source and render it in real time.
+The last icon at the top of the content editor lets you toggle hiding the
+content rendering area, so you can use the entire browser space to work on the
+HTML code. The enlarged HTML editor can really help you stay "in the zone" as
+you create awesome blog entries.
```
Comment (sez11a) [inline discover/portal/articles/07-collaborating/01-publishing-blogs.markdown]: Again, I don't think the benefit of this is the pop up window. I'm not even sure that it's bigger than what you get without popping up the window. The benefit here is the dual pane view.

File: discover/portal/articles/07-collaborating/01-publishing-blogs.markdown (hunk 1 above)
Comment (jhinkey) [inline discover/portal/articles/07-collaborating/01-publishing-blogs.markdown]: You're right. The main  benefit is the dual pane view. I just pushed up changes to reflect that. Thanks.

**Comment by sez11a** [comment]
Merged locally.

**Pull request closed without merging**